src/utils/runtime/src/logger.rs

```diff
@@ -359,7 +369,7 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
         let otel_tracer = {
             let runtime = tokio::runtime::Builder::new_multi_thread()
                 .enable_all()
-                .thread_name("risingwave-otel")
+                .thread_name("rw-otel")
```

Comment:
What does `otel` mean?

Reply:
It's the abbreviation of `opentelemetry`.